pkg/cmd/create/create_terraform.go

```diff
@@ -444,9 +446,13 @@ func (options *CreateTerraformOptions) createOrganisationGitRepo() error {
 		if err != nil {
 			log.Logger().Infof("Skipping jx install")
 		} else {
-			err = options.installJx(devCluster, clusterDefinitions)
-			if err != nil {
-				return err
+			if !options.Flags.SkipJxInstall {
```

Comment:
I am confused, why is this flag needed? The code seems to imply that `SkipTerraformApply` is already just about that, skipping the jx install. What's the semantic difference between `skip-terraform-apply` and  `skip-jx-install`? 

What's about the use of both flags in combination?

Would something like the following not be better:

```go
	if options.Flags.SkipTerraformApply || options.Flags. SkipJxInstall {
            log.Logger().Infof("Skipping jx install")
            return nil
	} 

        devCluster, err := options.findDevCluster(clusterDefinitions)
	if err != nil {
		return err
	} 
	
         err = options.installJx(devCluster, clusterDefinitions)
	if err != nil {
		return err
	}

       return nil
```

Reply:
SkipTerraformApply allows you to generate terraform but not apply it
SkipJxInstall (SkipInstallation) allows you to generate terraform, apply it, but then use something like `jx boot` to install the cluster

Reply:
The `skip-installation` flag for terraform is the same as the `skip-installation` when creating a cluster without terraform. If the description of the `skip-terraform-apply` options isn't clear then we could update that in a separate PR?

Reply:
Sure, I guess my point here is also about the imo quite twisted logic and nested conditionals. What I suggest is afaict semantically equivalent and easier to take in.